sequencer.c

```diff
@@ -147,6 +147,8 @@ static GIT_PATH_FUNC(rebase_path_refs_to_delete, "rebase-merge/refs-to-delete")
  * command-line.
  */
 static GIT_PATH_FUNC(rebase_path_gpg_sign_opt, "rebase-merge/gpg_sign_opt")
+/* save --committer-date-is-author-date */
```

Comment:
We don't have a comment like this for the other files either, so I think we shouldn't have it here either.  Just the comment above the whole block is enough here.